source/agora/network/NetworkManager.d

```diff
@@ -287,7 +288,8 @@ public class NetworkManager
 
         logInfo("Establishing connection with %s...", address);
         auto node = new NetworkClient(this.taskman, this.banman, address,
-            this.getClient(address, this.node_config.timeout.msecs),
+            null, //FIXME: temporary
+            //~ this.getClient(address, this.node_config.timeout.msecs), //FIXME: what's going on here?!
```

Comment:
This line is one of the most important of the module.
It instantiate a client to send requests to remote nodes.

Reply:
yes, that's why the tests are broken now.